src/app/clients/create-address/create-address.component.html

```diff
@@ -1,71 +1,71 @@
-
-      <div fxLayoutAlign="center">
-        <mat-card class="example-card">
-          <mat-card-header>
-          </mat-card-header>
-          <mat-card-content>
-            <form class="example-form" (ngSubmit)="onSubmit()" #f="ngForm">
-              <mat-form-field class="example-full-width">
-                <mat-select placeholder="Address Type" [(ngModel)]="addressTypeId" [ngModelOptions]="{standalone: true}" ngDefaultControl>
-                  <div>
-
-                    <mat-option *ngFor="let item of clientAddress?.addressTypeIdOptions" name="option" [value]="item?.id">{{item?.name}}</mat-option>
-                  </div>
-                </mat-select>
-              </mat-form-field>
-
-              <mat-form-field class="example-full-width">
-                <input matInput name="street" placeholder="Street" ngModel>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <input matInput name="addressLine1" placeholder="Address Line 1" ngModel>
-              </mat-form-field>
-
-
-              <mat-form-field class="example-full-width">
-                <input matInput name="addressLine2" placeholder="Address Line 2" ngModel>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <input matInput name="addressLine3" placeholder="Address Line 3" ngModel>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <input matInput name="city" placeholder="City" ngModel>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <mat-select placeholder="State" [(ngModel)]="addressStateId" [ngModelOptions]="{standalone: true}" ngDefaultControl>
-                  <div>
-
-                    <mat-option *ngFor="let item of clientAddress?.stateProvinceIdOptions" name="option" [value]="item?.id">{{item?.name}}</mat-option>
-                  </div>
-                </mat-select>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <mat-select placeholder="Country" [(ngModel)]="addressCountryId" [ngModelOptions]="{standalone: true}" ngDefaultControl>
-                  <div>
-
-                    <mat-option *ngFor="let item of clientAddress?.countryIdOptions" name="option" [value]="item?.id">{{item?.name}}</mat-option>
-                  </div>
-                </mat-select>
-              </mat-form-field>
-              <mat-form-field class="example-full-width">
-                <input matInput name="postalCode" placeholder="Postal Code" ngModel>
-              </mat-form-field>
-
-              <button mat-raised-button>Create</button>
-
-            </form>
-
-          </mat-card-content>
-        </mat-card>
-
-
-      </div>
-
-
-    </aw-wizard-step>
-  </aw-wizard>
+<div class="container">
```

Comment:
Please follow a single column layout. Refer accounting section. Also please add some space in between fields to ensure some clarity.